bin/metrics-per-process.py

```diff
@@ -208,6 +227,12 @@ def main():
     dest    = 'graphite_scheme',
     metavar = 'GRAPHITE_SCHEME')
 
+  parser.add_option('-m', '--metrics',
```

Comment:
I think that a name like this: https://github.com/sensu-plugins/sensu-plugins-redis/blob/2.2.1/bin/metrics-redis-graphite.rb#L94-L98 would be better

Reply:
Sure. `add_option(-k, --filter-keys, ..., 'keys', 'KEYS_REGEXP')`?

Reply:
Is this an inclusive or exclusive filter? If inclusive I think `--keys-regex` otherwise `--skip-keys-regex` but I do not feel super strongly about it. @eheydrick ping

Reply:
That or `--metrics-regex` given that metrics is the thing being collected and more specific.

Reply:
I'd be :+1:  with `--metric-regexes` to make it obvious that we are not requiring a single regex but a list of regexes to be used.

Reply:
But it's actually a single regex, isn't it? Or do we want to have a comma-separated list of regexes to make it easier for the user to write the regex? IMO, separating keys with a `|` in a single regex isn't more difficult or very different  from several regexes separated with a `,`.

Reply:
that is true when you have very simple regexes but the more complex the more it is a pain to look at. I'd prefer a comma separated list of regexes as this follows established patterns in both graphite and redis plugins. If we think that is overkill I am ok with limiting the scope but prefer consistency where possible.

Reply:
Alright!